packages/vitest/src/integrations/css/css-modules.ts

```diff
@@ -0,0 +1,19 @@
+import type { CSSModuleScopeStrategy } from '../../types'
+
+export function generateCssFilenameHash(filename: string) {
+  return Buffer.from(filename).toString('base64').substring(0, 6)
```

Comment:
We need to normalize path separators so snapshots created on linux will still work on windows

Reply:
We are using `relative` from pathe which normilizes it already (you can see out Windows tests are passing)